docs/Actions/sanitize-html.md

```diff
@@ -0,0 +1,118 @@
+
+## `Don't use it to`
+
+- Sanitize multiple inputs at once. Use a separate action for each input.
+- To prevent SQL injection. Use Bind Tokens in your [Run SQL Query](/docs/Actions/run-sql) action instead.
+
+## `Related Actions`
+
+| Action Name                                     | Description                                                                        |
+| ----------------------------------------------- | ---------------------------------------------------------------------------------- |
+| [Generate PDF](/docs/Actions/generate-pdf.md)   | Uses wkhtmltopdf open source to generate PDF from HTML template.       |
+| [Send Email](/docs/Actions/send-email.md)       | Sends an email to one or multiple recipients. |
+| [Run SQL Query](/docs/Actions/run-sql-query.md) | Executes an SQL statement and captures the output.                                 |
+
+
+## `Input Parameter Reference`
+
+| Parameter                 | Description                                                                                                                 | Supports Tokens | Default        | Required |
+| ------------------------- | --------------------------------------------------------------------------------------------------------------------------- | --------------- | -------------- | -------- |
+| Html Content | The HTML content to be sanitized. Most often this will be a token from the input source.                                                 | Yes             | `empty string` | No       |
+| Convert links to nofollow      | Ensures all anchor tags in the provided HTML content have the "rel" attribute value set to "nofollow". | No             | false              | No       |
+| Removable attributes      | The attributes specified here will be stripped off the HTML tags that were allowed in the provided HTML. Enter one at a time and press enter. | No             | `empty string`              | No       |
+| Disable scripts      | Removes all script tags and their content from the provided HTML. | No             | false              | No       |
+| Toggle between Exclude/Allow HTML tag lists      | Applies to the Excluded Tags option below. When this option is checked Allow Mode is in effect.   | No             | Exclude             | No       |
+| Excluded Tags      | HTML tags specified here will be removed from the provided content. All other tags will remain in place. Content between the removed tags will also remain   | No             | `empty string`               | No       |
+| Allowed Tags      | Only HTML tags specified here will be preserved. All other tags will be deleted and their content processed recursively. | No             | `empty string`               | No       |
+| Allowed Tags      | Only HTML tags specified here will be preserved. All other tags will be deleted and their content processed recursively. | No             | `empty string`               | No       |
```

Comment:
This seems a duplicate line. Also, I'm not sure if users would  understand what "content processed recursively".

Reply:
It was supposed to be the Removable Tags section. Updated in latest commit